pkg/juicefs/mount/pod_mount.go

```diff
@@ -315,9 +316,14 @@ func (p *PodMount) genMountPodName(ctx context.Context, jfsSetting *jfsConfig.Jf
 func (p *PodMount) createOrAddRef(ctx context.Context, podName string, jfsSetting *jfsConfig.JfsSetting, appinfo *jfsConfig.AppInfo) (err error) {
 	klog.V(6).Infof("createOrAddRef: mount pod name %s", podName)
 	jfsSetting.MountPath = jfsSetting.MountPath + podName[len(podName)-7:]
-	jfsSetting.SecretName = podName + "-secret"
+	secretHandle := jfsSetting.VolumeId
+	if jfsConfig.StorageClassShareMount {
+		secretHandle = jfsSetting.UniqueId
+	}
```

Comment:
use `secretHandle := jfsSetting.UniqueId` 

If not storageClassShareMount, jfsSetting.UniqueId is volumeId.